src/Compilers/CSharp/Test/Semantic/Semantics/RecordStructTests.cs

```diff
@@ -5697,7 +5696,6 @@ private class AnalyzerActions_01_Analyzer : DiagnosticAnalyzer
         {
             public int FireCount0;
             public int FireCountRecordStructDeclarationA;
-            public int FireCountRecordStructDeclarationACtor;
```

Comment:
Consider keeping the integer but assert against the new expected value.

Reply:
@Youssef1313 After this change, `FireCountRecordStructDeclarationACtor` has no writes and would always have its default value. We do assert that the default case in the switch is never encountered.